packages/create-block/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@wordpress/create-block",
+  "version": "0.5.1",
```

Comment:
How'd you land on this as the initial version? Inherited from `create-wordpress-block` ? Should it be?

Reply:
Yes, I copied all files over from `create-wordpress-block`. I planned to use `0.6.0` in the next release. It can be also `0.1.0`. I don't care as much to be honest.